packages/ckeditor5-ui/tests/bindings/addkeyboardhandlingforgrid.js

```diff
@@ -275,6 +275,72 @@ describe( 'addKeyboardHandlingForGrid()', () => {
 		} );
 	} );
 
+	describe( 'arrows moves in rtl', () => {
+		let view, keystrokes, focusTracker, gridElementsCollection, numberOfColumnsStub, uiLanguageDirection;
```

Comment:
You don't really need any of those variables:

*   `view, keystrokes, focusTracker, gridElementsCollection` are all already in the scope so you can reassign them.
*   `numberOfColumnsStub, uiLanguageDirection` have always the same values in your tests, so these values can be used directly.